app/views/admin/rdvs_users/update.js.erb

```diff
@@ -0,0 +1,17 @@
+<% if @rdvs_user.errors.empty? %>
+  <% participation_dropdown_html = render "admin/rdvs_users/participation_status_dropdown", rdvs_user: @rdvs_user, agent: @agent %>
+  <% participants_count_html = render "admin/rdvs/participants_count", rdv: @rdv %>
+  (function() {
+    let participation_dropdown = document.getElementById('participation-dropdown-<%= @rdvs_user.id %>');
+    if(participation_dropdown) {
+      participation_dropdown.outerHTML = "<%= escape_javascript participation_dropdown_html %>";
+    }
+    let participants_count = document.getElementById('participants-count-<%= @rdv.id %>');
+    if(participants_count) {
+      participants_count.outerHTML = "<%= escape_javascript participants_count_html %>";
+    }
+    $("#participation-flash-message-<%= @rdvs_user.rdv.id %>").html('<%= j render partial: "layouts/flash" %>');
+  })();
+<% else %>
+    $("#participation-flash-message-<%= @rdvs_user.rdv.id %>").html('<%= j render partial: "layouts/flash" %>');
```

Comment:
Je sais qu'à l'époque avec Nicolas il y avait la discussion de ne plus renvoyer du JS sur les nouvelles vues mais du HTML over the wire 😄 , mais je sais pas si c'est toujours d'actualité. Peut-être que @yaf sait ?

Reply:
Pour info on en a un peu discuté ce jour en réunion tech. On en reparlera quand Victor, Yannick et toi seront présent mais l'accueil de stimulus + turbo au sein de l'équipe est plutôt favorable. On adaptera ce code avec du stimulus le moment venu dans une PR dédiée.